### PR TITLE
PUBDEV-5504  Another mode to treat missing values: plug values

### PR DESCRIPTION
**Pull request by michalkurka**
Just use means but add another field to represent it, should pass all tests

### CODE REVIEW

File: h2o-algos/src/test/java/hex/glm/GLMBasicTestRegression.java [hunk 1]
```diff
@@ -134,7 +134,7 @@ public void testSparse() {
     parms._objective_epsilon = 0;
     parms._gradient_epsilon = 1e-10;
     parms._max_iterations = 1000;
-    parms._missing_values_handling = DeepLearningModel.DeepLearningParameters.MissingValuesHandling.Skip;
+    parms._missing_values_handling = GLMModel.GLMParameters.MissingValuesHandling.Skip;
     try {
```
Comment (wendycwong) [inline h2o-algos/src/test/java/hex/glm/GLMBasicTestRegression.java]: OMG:  This is really bad!  Thanks for fixing this one.

File: h2o-algos/src/main/java/hex/glm/GLMModel.java [hunk 2]
```diff
@@ -330,8 +337,14 @@ public void validate(GLM glm) {
             H2O.fail();
         }
       }
+      if (_missing_values_handling != null) {
+        if (!(_missing_values_handling instanceof DeepLearningModel.DeepLearningParameters.MissingValuesHandling) &&
+                !(_missing_values_handling instanceof MissingValuesHandling)) {
```
Comment (wendycwong) [inline h2o-algos/src/main/java/hex/glm/GLMModel.java]: the !(_missing_values_handling instanceof DeepLearningModel.DeepLearningParameters.MissingValuesHandling) part is for backward compatibility, right?

Comment (michalkurka) [reply h2o-algos/src/main/java/hex/glm/GLMModel.java]: exactly!

**Comment by wendycwong** [comment]
Thank you for doing this and fixing errors along the way.

**Pull request merged**